dashboards-notebooks/.cypress/integration/ui.spec.js

```diff
@@ -161,20 +161,6 @@ describe('Testing paragraphs', () => {
     cy.get('td').contains('b2').should('exist');
   });
 
-  it('Shows output message', () => {
```

Comment:
why are we removing this?

Reply:
UX feedback was to remove the "Output below is stale/Output is up-to-date" text and only include the "Last successful run: <timestamp>" string

Reply:
can you update strings to "Last successful run:" rather than removing the test? also is the `div[style*="opacity: 0.5"]` opacity style removed too?

Reply:
👍